README.md

```diff
@@ -23,7 +23,7 @@ and comes with a JavaScript-based scoreboard to display team rankings.
 Running a Development Server
 ============================
 
-    docker run --rm -it -p 8080:8080 dirtbags/moth-devel
+    docker run --rm -it -p 8080:8080 -v /path/to/puzzles:/puzzles:ro dirtbags/moth-devel
```

Comment:
My intent here was that the devel server should start up with the built-in puzzles unless you override it. So I feel like this should actually be a new recipe under "Using your own puzzle categories" in the Devel server section.